### ISSUE TITLE
One loss or two loss functions?

### ISSUE THREAD

**Issue by CassieMai** [ZhihengCV/Bayesian-Crowd-Counting#5]
There is two loss in your model, right? One is the Bayesian loss for point supervision and the other for training the D^est. Is the loss for training D^est the pixel-wise loss?

One more question is about p(y_n|x_m). The value of y_n varies from one image to another image. That is, N is different for each image. The N of a test image is also different from that of training images. What is the value field of y_n in training?

**Comment by xingxjtu** [comment]
There is only one loss function (Bayesian loss) to estimate D^est. Each sample has a specific value field of y_n.
